2018-11/index.md

```diff
@@ -4,69 +4,69 @@ title: BeagleBoard.org Enables Success
 events:
- - title: World MakerFaire Education Forum, New York City
-   date: September 21, 2018 
-   link: https://beagleboard.org/blog/2018-09-05-join-beagleboard-org-at-world-maker-faire-in-new-york-sept-21-23
- - title: BeagleBoard.org 10th Anniversary &num;BringAHack Event, New York City
-   date: September 21, 2018 
-   link: https://www.eventbrite.com/e/bringahack-new-york-with-beagleboardorg-before-maker-faire-tickets-49521195228
- - title: World MakerFaire, New York City
-   date: September 22-23, 2018
-   link: https://beagleboard.org/blog/2018-09-05-join-beagleboard-org-at-world-maker-faire-in-new-york-sept-21-23
+ - title: Embedded World 2019 Exhibition & Conference, Nuremberg, Germany
+   date: February 26-28, 2019 
+   link: https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-38827067/beagleboard-org-foundation
+ - title: Linux Jumpstart with PocketBeagle Hands-On Workshop at Embedded World
+   date: February 27, 2019 
+   link: https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-38827067/beagleboard-org-foundation
+ - title: Want to schedule a meeting with Jason or Christine?
+   date: Schedule online
+   link: https://beagleboard.org/about
```

Comment:
```suggestion
   link: https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-38827067/beagleboard-org-foundation#exhibitorcontact
```

At least use the direct calendly link to try to save folk's time. I need to explore if I should update calendly.com, but I think maybe we should just use Embedded World's system. I just sent a test message.

Reply:
sounds good